docs/modules/ROOT/pages/erc1155-burnable.adoc

```diff
@@ -0,0 +1,50 @@
+= ERC-1155 Burnable
+
+Extension of xref:erc1155.adoc[ERC-1155] that allows token holders to destroy both their
+own tokens and those that they have been approved to use.
+
+[[usage]]
+== Usage
+
+In order to make https://docs.rs/openzeppelin-stylus/0.2.0/openzeppelin_stylus/token/erc1155/extensions/burnable/index.html[`ERC-1155 Burnable`] methods “external” so that other contracts can call them, you need to implement them by yourself for your final contract as follows:
```

Comment:
There may be sth wrong with version in the link (CI checks) as it does not exist.

Reply:
I noticed some of the other adocs were using links similar to this (only using 0.1.1 in the URL instead of 0.2.0), maybe I got ahead of myself here. Should I just remove the link for now?

Reply:
Make it `v0.2.0-alpha.2` and let's check if CI jobs passes.